inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -39,17 +39,23 @@
           <h1 class="record-detailed-title">{{ render_record_title(record) }}</h1>
           <div id="record-authors">
             {{ render_record_authors(record, is_brief=false, number_of_displayed_authors=25) }}
-            <span id="record-date">- {{ record.get('earliest_date')|format_date }}</span>
+            <span id="record-date">
+            {% if record.get('authors') or record.get('collaborations') %}
+            - {{ record.get('earliest_date')|format_date }}
+            {% else %}
+              {{ record.get('earliest_date')|format_date }}
```

Comment:
Why is the `if-else` needed if in the end the same code gets executed?

Reply:
(It's the dash!)